### ISSUE TITLE
Is there an exe to do the copy paste trick on windows ?

### ISSUE THREAD

**Issue by antoninadert** [eXpl0it3r/bnkextr#5]
Question is in the title

**Comment by eXpl0it3r** [comment]
Can you rephrase the question, as I don't know what you mean.

Release builds are in the release tab. 🙂

**Comment by antoninadert** [comment]
I think that helps. I just wondered if there was a way to drag-drop or copy-paste the BNK file on the .exe to actually extract it ?

**Comment by eXpl0it3r** [comment]
Ah, now I get what you mean. No, you just open a command prompt and call the exe.
But I can add it as feature request. 🙂

**Comment by antoninadert** [comment]
That would be a good feature :) , 👍

**Comment by DavidCodesDev** [comment]
> That would be a good feature :) , 👍

You can use this: https://github.com/Vextil/Wwise-Unpacker
It uses eXpl0it3r's code to extract the audio files from the bnk files, all you have to do is to drag and drop all your bnk's in ''Game Files'' folder and run the .bat file :)

**Comment by abheekda1** [comment]
I believe dragging and dropping the file on the EXE just passes the path as a parameter does it not?
